### PR TITLE
Update some information

### PR DESCRIPTION
**Pull request by slotThe**
To get things rolling, here are some things I noticed while strolling through the new site: 

1. Clean up community.md

  - The blog is no longer active, remove it
  - Reorder community items so our most-used forms of communication are
    near the top
  - Add information to join the IRC channel via webchat and that it's
    linked to matrix
  - Differentiate bug reports to xmonad and xmonad-contrib

2. Update distribution links

  - Remove dead links, update the NetBSD one

3. Update the line number statement

  - Because why not :)

### CODE REVIEW

**Comment by eyenx** [comment]
LOL lots of trailing whitespaces.

**Review by eyenx — APPROVED** [review]

**Review by TheMC47 — APPROVED** [review]
Awesome! Thanks

File: community.md [hunk 1]
```diff
@@ -7,11 +7,12 @@ title: Community
 
 ### The xmonad community
 
+*   [the irc channel](http://haskell.org/haskellwiki/IRC_channel): `#xmonad@chat.freenode.org` (join it via [webchat](https://webchat.freenode.net/))
+*   [the matrix channel](https://matrix.to/#/#freenode_#xmonad:matrix.org): this is linked to the above IRC channel
```
Comment (liskin) [inline community.md:11]: I wouldn't call this **the** matrix channel, as this is in fact still very much "the irc channel", just through a bridge. There *is* **a** separate matrix room (channel) `#xmonad:matrix.org` which isn't very active and which could possibly later be linked to the IRC bridged one, but it's not right now, but it's the first (and possibly only) one that appears when searching for xmonad. So a more correct wording would be something like "join it via webchat or matrix" or a separate bullet point named "matrix bridge to the irc channel".

**Comment by liskin** [comment]
> LOL lots of trailing whitespaces.

Maybe that's because two trailing spaces in markdown mean `<br>`? Most of those were in bullet points, though, so this was either pointless or downright harmful, but perhaps that's where it originated from?

**Comment by TheMC47** [comment]
> Most of those were in bullet points, though, so this was either pointless or downright harmful

Yep. Yikes. I'll fix it

**Comment by TheMC47** [comment]
> perhaps that's where it originated from

These weren't bullets, just lines of text

**Comment by liskin** [comment]
> These weren't bullets, just lines of text

Oh, right, my eyes are so tired I can't read diffs any more. Sorry. :-D

**Comment by slotThe** [comment]
On Sat, Apr 17 2021 15:12, Tomáš Janoušek wrote:
> I wouldn't call this **the** matrix channel, as this is in fact still
> very much "the irc channel", just through a bridge.

I think it's fair to say it's _the official_ xmonad matrix server.  That
it happens to be bridged to some IRC channel seems secondary to me.

**Comment by liskin** [comment]
> I think it's fair to say it's _the official_ xmonad matrix server. That it happens to be bridged to some IRC channel seems secondary to me.

I don't necessarily agree. It happens to be the matrix room where the devs hang out, and that is more active, but one could argue that `#xmonad:matrix.org` is "_the_ matrix room", not `#freenode_#xmonad:matrix.org`. Obviously if these weren't separate rooms, we wouldn't even need to have this disccusion, and everything would be better. :-/

So anyway, now I think the best wording (or just more accurate) is just:

`[a matrix room](https://matrix.to/#/#freenode_#xmonad:matrix.org) bridged to the above IRC channel`

**Comment by slotThe** [comment]
On Sun, Apr 18 2021 03:18, Tomáš Janoušek wrote:
> I don't necessarily agree. It happens to be the matrix room where the
> devs hang out, and that is more active, but one could argue that
> `#xmonad:matrix.org` is "_the_ matrix room", not
> `#freenode_#xmonad:matrix.org`. 

I feel like just because the former sounds more official doesn't
necessarily make it so.  Making where the devs hang out the official
channel seems like the sane thing to do to me; unless, perhaps, there's
already a big community somewhere else.  I remember `#xmonad:matrix.org`
being not-at-all active, so that doesn't seem to be a factor.

It would be a different story if we controlled both of these rooms and
one was specifically for development and one for asking questions or
something like that but, alas, that's not the case here :/

> Obviously if these weren't separate rooms, we wouldn't even need to
> have this disccusion, and everything would be better. :-/

Indeed; maybe we'll be able to make contact with the owner of the other
channel at some point...

> So anyway, now I think the best wording is just:
>
> `[a matrix room](https://matrix.to/#/#freenode_#xmonad:matrix.org)
> bridged to the above IRC channel`

Sure, sounds good to me!

**Comment by liskin** [comment]
> Indeed; maybe we'll be able to make contact with the owner of the other channel at some point...

Well it's just a matter of doing it, they were responsive the last time I had to communicate with them. :-)

(So it's just that I have other more pressing stuff to do. :-/)

**Comment by slotThe** [comment]
On Sun, Apr 18 2021 03:45, Tomáš Janoušek wrote:
> Well it's just a matter of doing it, they were responsive the last
> time I had to communicate with them. :-)
>
> (So it's just that I have other more pressing stuff to do. :-/)

Oh!  I thought they were unresponsive, sorry for the confusion.  I
suppose other people with matrix accounts could do that if they have the
time @psibi @themc47 (sorry to pick you out specifically, but you're the
only two who I've seen use matrix :))

**Comment by TheMC47** [comment]
Sure, I can get in touch. But what should the contact be? What do we want?

**Comment by liskin** [comment]
> Sure, I can get in touch. But what should the contact be? What do we want?

First, we'll need to find out if it's possible to use the official Matrix freenode bridge for this, or if bridging to a custom-named room requires someone to run a separate instance of the IRC bridge. It might be documented somewhere, or it may be easier to just ask the bridge ops. Start here: https://matrix.org/docs/projects/bridge/matrix-org-irc-bridge

Then, the admin/owner of the `#xmonad:matrix.org` room is `@unclechu:matrix.org`, and the question is whether they think bridging the IRC channel into the `#xmonad:matrix.org` room is desired or whether they'd prefer to keep it quiet and separate.

**Comment by psibi** [comment]
> It happens to be the matrix room where the devs hang out, and that is more active, but one could argue that #xmonad:matrix.org is "the matrix room", not #freenode_#xmonad:matrix.org.

I never know about the matrix room, I always assumed that the bridge one was the only one present. Anyway, I have joined there too.

> I suppose other people with matrix accounts could do that if they have the time @psibi @TheMC47 (sorry to pick you out specifically, but you're theonly two who I've seen use matrix :))

No worries. It seems @TheMC47 is top of that (Thanks for that and all the other work you have been doing!).

**Comment by TheMC47** [comment]
> Thanks for that and all the other work you have been doing!

Just doing my part!

I sent them a private message, we'll wait and see now

**Comment by liskin** [comment]
> I sent them a private message, we'll wait and see now

Has there been any news? Freenode vs Matrix vs Libera vs … seems to be a hot topic now, might be a good opportunity to make this happen. :-)

**Comment by TheMC47** [comment]
> Has there been any news? Freenode vs Matrix vs Libera vs … seems to be a hot topic now, might be a good opportunity to make this happen. :-)

Sorry, I thought I posted the feedback. They were ready to work, they even tried doing it before but failed. Anyway, if someone is up to try it, they're willing to co-operate. I don't think I'll have some free time soon

**Comment by liskin** [comment]
Oh, you probably did, these words you just wrote seem vaguely familiar now. :-)
Probably sent that to IRC and I looked for it here instead.
I'll see what I can do then.

**Comment by geekosaur** [comment]
Yes, I recall that IRC discussion and then our fixing the channel flags so it would work.

**Comment by liskin** [comment]
I think those were two separate discussions actually. And two separate Matrix rooms. It keeps being confusing, which is why I'd really like to unconfuse it.

**Pull request merged**